taichi/common/interface.h

```diff
@@ -176,7 +176,7 @@ class InterfaceHolder {
              implementation_factories.end();                                  \
     }                                                                         \
     void remove(const std::string &alias) override {                          \
-      assert_info(has(alias),                                                 \
+      TI_ASSERT_INFO(has(alias),                                                 \
                   std::string("Implemetation ") + alias + " not found!");     \
       implementation_factories.erase(alias);                                  \
     }                                                                         \
```

Comment:
The `\` on the right is not aligned... Will auto-format resolve this?

Reply:
You are right - auto-formatting is our friend...